frontend/entrypoints/sidepanel/App.module.css

```
@@ -0,0 +1,293 @@
+
+.search input:focus {
+  border-color: var(--color-action);
+  outline: 3px solid var(--color-focus);
+}
+
+.visuallyHidden {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)